src/components/blog/ProgressBar.tsx

```
@@ -0,0 +1,34 @@
+}
+
+const getColorClass = (status) => {
+  switch (status) {
+    case 1:
+      return 'bg-[#FFB47D]' // 1개
+    case 2:
+      return 'bg-[#FFA451]' // 2개
+    case 3:
+      return 'bg-[#FF6B00]' // 3개 이상
+    case 0:
+    default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)